client/devdocs/controller.js

```diff
@@ -14,13 +14,10 @@ import url from 'url';
  * Internal dependencies
  */
 import config from 'config';
+import AsyncLoad from './async-load';
```

Comment:
This would've been a bit easier to follow for me if it was named `DevdocsAsyncLoad` – when reading the component below I thought it was using the generic `AsyncLoad`.

Reply:
That's a good point–I'll make the change! I started now but it's late and for some reason I'm getting a fair number of build errors where I didn't used to be. I'll check it again tomorrow morning 😄

Reply:
if we have to create a new async-loader we could also update the babel transform to resolve the hard-coded-naming-issue